pulp_rpm/app/serializers/custom_metadata.py

```diff
@@ -19,12 +19,14 @@ class RepoMetadataFileSerializer(
     data_type = serializers.CharField(help_text=_("Metadata type."))
     checksum_type = serializers.CharField(help_text=_("Checksum type for the file."))
     checksum = serializers.CharField(help_text=_("Checksum for the file."))
+    relative_path = serializers.CharField(help_text=_("Relative path of the file."))
 
     class Meta:
         fields = SingleArtifactContentUploadSerializer.Meta.fields + (
             "data_type",
             "checksum_type",
             "checksum",
             "sha256",
+            "relative_path"
```

Comment:
What is the user experience here?

Does it mean the relative path is visible now in REST API when listing the content?
What happens at upload time? Do we request anything new?

Reply:
It won't, this part removed.

Reply:
Yet, I left out the last comma (,) as not needed with last member of tuple/list.